include/rdmini/vandercorput.h

```diff
@@ -0,0 +1,86 @@
+#ifndef VANDERCORPUT_H_
+#define VANDERCORPUT_H_
+
+#include <iostream>
+#include <random>
+#include <limits> 
+
+//-----------------------------------------------------------------------------
+namespace rdmini{
+class Linear_RNG {
```

Comment:
For consistency with other names in rdmini, I would use a lower case name. Also, just to make it clear that this is a very non-random generator, perhaps it could be called `sequential_generator` or `counting_generator` instead?